packages/odyssey-react/src/components/FieldGroup/FieldGroup.tsx

```diff
@@ -26,9 +26,9 @@ export interface FieldGroupProps {
   children: ReactElement | ReactElement[];
 
   /**
-   * The title of the FieldGroup.
+   * The legend of the FieldGroup.
    */
-  title?: string;
+  legend?: string;
```

Comment:
I didn't see a `BREAKING CHANGE` or `!` commit title to describe this change we should call it out explicitly!

Reply:
Commits updated to call out all unique changes. I've grouped the simpler "title" -> "heading" changes under one commit with affected components listed in the body.